src/interventions.c

```diff
@@ -1219,12 +1366,21 @@ void intervention_on_traced(
 
 	parameters *params = model->params;
 
+	int lateral_flow_test = params->lateral_flow_test_on_traced && gsl_ran_bernoulli( rng, params->lateral_flow_test_fraction );
```

Comment:
If `params->lateral_flow_test_on_traced=FALSE` does the gsl_ran_bernoulli() get called?
Probably best to move to inside the if statement below.